### PR TITLE
Cors fix for heroku

### PR DESCRIPTION
**Pull request by croixk**
What changed?
- added several changes to fix cors issues
Why is this change necessary?
- FE was not able to access our BE link 
What changed technically that may impact others?
- this should not impact existing code 
How do we test it?
- we did preliminary testing by accessing the BE heroku link and in postman

### CODE REVIEW

File: config/initializers/cors.rb [hunk 1]
```diff
@@ -6,11 +6,12 @@
...
-#   allow do
-#     origins 'example.com'
-#
-#     resource '*',
-#       headers: :any,
-#       methods: [:get, :post, :put, :patch, :delete, :options, :head]
-#   end
+  # allow do
+  #   # origins 'example.com'
+  #   origins '*'
```
Comment (kbhoffmann) [inline config/initializers/cors.rb:11]: Ok great, I was wondering about this!

File: config/routes.rb [hunk 2]
```diff
@@ -4,4 +4,6 @@
     mount GraphiQL::Rails::Engine, at: "/graphiql", graphql_path: "/graphql#execute"
   end
   post "/graphql", to: "graphql#execute"
+  mount ActionCable.server, at: '/cable'
```
Comment (kbhoffmann) [inline config/routes.rb:7]: Interesting, what is this for?

**Review by kbhoffmann — APPROVED** [review]
Nice work being persistent with this Croix!

**Review by omegaeye-he — COMMENTED** [review]
LGTM

**Pull request merged**